added "pitch" to readme (#112)
ADAH-EviDENce/evidence: README.md

# evidence

> Doc2Vec-based assisted close reading tool with support for abtract concept-based search and context-based search.

Reviewer: ```suggestion
Doc2Vec-based assisted close reading tool with support for abstract concept-based search and context-based search.
```

> ## Machine supported research in humanities

Reviewer: ```suggestion
## Machine-supported research in humanities
```

> While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.

Reviewer: ```suggestion
While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.
```

> What this boils down to, is the fact that it is often restrictive and difficult to formulate selection criteria, in particular for more complex or abstract concept, in the framework of a keyword based search which is the standard entry point to digitized text collections.

Reviewer: ```suggestion
What this boils down to, is the fact that it is often restrictive and difficult, if not impossible, to formulate adequate selection criteria, in particular for more complex or abstract concepts, in the framework of a keyword based search which is the standard entry point to digitized text collections.
```

## Querying by example - close reading with tailored suggestions  

> Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.

Reviewer: One could mistake "Evidence" for its normal meaning. Can you think of a way to avoid this confusion?

Author: how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Author: @jspaaks how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Reviewer: Whichever way you think is OK, I just wanted to raise the issue. Simplest way to resolve this is maybe just ``evidence``?

Author: I was thinking the same. I'll go for that

> Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.

Reviewer: is it word2vec or doc2vec? either way we should pick one and use it consistently throughout.

Author: it is doc2vec in all cases. I'll give it a comb through

Reviewer: ```
$ grep -inr word2vec
Binary file experiments/demo/model/doc2vec.d2v matches
README.md:3:Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
README.md:50:## Generating the word2vec model
```

Author: thanks

> Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.

Reviewer: ```suggestion
Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework, evidence learns abstract representations of the content of the elements of the user's corpus.
```

> Departing from a set of corpus elements that the user selects as relevant starting points evidence retrieves similar elements and presents them to the user, using the users feedback to refine its retrieval.

Reviewer: ```suggestion
Departing from a set of corpus elements that the user selects as relevant starting points, evidence retrieves similar elements and presents them to the user, using the users feedback to refine its retrieval.
```

> This enables a user to combine the power of a close-readiing approach with that of a large digitized corpus, selecting elements from the entire corpus which are likely to be of interest, but leaving the dcision up to the user.

Reviewer: ```suggestion
This enables a user to combine the power of a close-reading approach with that of a large digitized corpus, selecting elements from the entire corpus which are likely to be of interest, but leaving the decision up to the user.
```
